tests/test.js

```diff
@@ -97,7 +99,9 @@ async function startRequestProfiling(page) {
     const numberOfUrlRequests = requestTimes[url] ?? 0;
     requestTimes[url] = numberOfUrlRequests + 1;
     if (numberOfUrlRequests !== 0) url = `${url}_${numberOfUrlRequests}`;
-    requests[url] = request.timing();
+    requests[url]["timing"] = request.timing();
+    const response = await request.response();
+    requests[url]["headers"] = response.headers();
```

Comment:
I noticed, `Content-Length` header is sometimes missing. Would it be possible to add response (body) size ?